src/main/resources/resource/Arduino/generate/arduinoMsgs.schema

```diff
@@ -141,5 +141,5 @@
 # Amt203=0 / As5048A=1 Encoder methods
 > encoderAttach/deviceId/type/pin
 > setZeroPoint/deviceId
-< publishEncoderPosition/deviceId/b16 position
+< publishEncoderData/deviceId/b16 position
```

Comment:
I didn't see the MrlComm version number get incremented with this change..  seems like we should bump the version if we've made a chance like this.

Reply:
Oooh definitely ...  nice catch !